app/views/courses/fee_or_salary/_form_fields.html.erb

```diff
@@ -7,21 +7,11 @@
       <h1 class="govuk-fieldset__heading">
         Is it fee paying or salaried?
       </h1>
+      <br>
```

Comment:
`<br />` shurely

Reply:
![image](https://user-images.githubusercontent.com/42515961/69722347-c032f200-110e-11ea-87f3-e62639ec0f30.png)
Guess it depends 🤷‍♂

Reply:
oh okay that was xhtml, it's fine in html5, encouraged in fact